src/glossary.js

```diff
@@ -89,21 +93,48 @@ function Glossary(terms, selectors) {
   this.addEventListener(document.body, 'keyup', this.handleKeyup.bind(this));
 }
 
-/** Populate internal list.js list of terms */
 Glossary.prototype.populate = function() {
+  var self = this;
+  var terms = this.terms.map(function(term) {
+    return {
+      'term': term.term,
+      'definition': term.definition,
+      'termClass': self.classes.termClass,
+      'definitionClass': self.classes.definitionClass
+    };
+  });
+
+  var itemTemplate = _.template(
+    '<li>' +
+      '<button class="data-glossary-term {{ termClass }}">{{ term }}' +
+      '</button>' +
+      '<div class="{{ definitionClass }}">{{ definition }}</div>' +
+    '</li>',
+    {interpolate: /\{\{(.+?)\}\}/g}
+  );
+
+  terms.forEach(function(term) {
+    this.list.insertAdjacentHTML('beforeend', itemTemplate(term));
+  }, this);
+};
+
+/** Initialize list.js list of terms */
+Glossary.prototype.initList = function() {
+  var glossaryId = this.selectors.body.slice(1);
```

Comment:
Well, I specified in the README that those are what they should be. Is there a more fail-proof way that would be better?